src/App.css

```diff
@@ -5,7 +5,6 @@
   margin: 0 auto;
```

Comment:
with the current css the `main` element is overflow the nav element and footer and you can see the scrollbar through entire page
<img width="1427" alt="image" src="https://github.com/user-attachments/assets/108c769f-fca4-40fe-b1b8-9cea855dc736">

Reply:
Disabling all these root styles didn't help to rid off scroll bar overflow. I would appreciate any help with styling to fix.

Reply:
min-height: 100vh; -> this causes overflow, But I need it